build.assets/makefiles/etcd/etcd-upgrade.service

```diff
@@ -18,14 +18,15 @@ EnvironmentFile=-/ext/etcd/etcd-version.txt
 # Set TLS ciphers as per mozilla recommendations
 # https://wiki.mozilla.org/Security/Server_Side_TLS
 Environment=ETCD_CIPHER_SUITES=TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+UnsetEnvironment=ETCD_INITIAL_CLUSTER_STATE ETCD_INITIAL_CLUSTER
 ExecStartPre=/usr/bin/planet etcd init
 ExecStart=/usr/bin/etcd \
         --name=${PLANET_ETCD_MEMBER_NAME} \
         --data-dir=/ext/etcd/${PLANET_ETCD_VERSION} \
         --initial-advertise-peer-urls=https://${PLANET_PUBLIC_IP}:2380 \
         --advertise-client-urls=https://127.0.0.2:2379,https://127.0.0.2:4001 \
         --listen-client-urls=https://127.0.0.2:2379,https://127.0.0.2:4001 \
-        --listen-peer-urls=https://${PLANET_PUBLIC_IP}:2380,https://${PLANET_PUBLIC_IP}:7001 \
+        --listen-peer-urls=https://127.0.0.2:2380,https://127.0.0.2:7001 \
```

Comment:
Yea, to my knowledge we don't need the additional ports, I just sort of left them because I didn't know why it was configured this way in the first place. Last time I went through, I think all client comms are correctly using 2379 only, and server to server comms only need the one port, which 2380 should be fine for.

Would also allow us to simplify the requirements in our docs ever so slightly.